### ISSUE TITLE
full transfer from orcbrew files

### ISSUE THREAD

**Issue by The60Gunner** [Orcpub/orcpub#434]
**Motivation / Use Case**

i think somethings amiss with DMV and importing from Orcbrew files, i have an AllContent Orcbrew file with my characters/items/spells ect. and it only transfers my spells, feats, races, classes but not the items and characters themselves.

**Expected Behavior**

i humbly request that orcbrew files load All Content in them so i can have all the items i created without having to recreate them all over again, and i would like to see an import option for characters, all i see is NEW, RANDOM, CLONE, SAVE NEW CHARACTER, EXPORT

**Other Information**

basically orcbrew files load all content including characters/items and an import characters ability

if these already exist then i havent seen it anywhere but i know the import doesnt import everything.

**Comment by DivertedCircle** [comment]
Characters and items are saved to your account. Everything else; races, feats, classes, etc, are saved to orcbrew files. You can log into DMV using your account on any machine to access your characters and items, but you have to upload your orcbrew file on each new machine. This is to prevent copyright issues due to hosting non-SRD content on the website.

**Comment by datdamnzotz** [comment]
Functionality exist if you host your own

See https://github.com/Orcpub/orcpub#importing-your-orcbrews-automatically
